slider/replay.py

```diff
@@ -576,6 +636,62 @@ def parse(cls, data, *, library=None, client=None, save=False):
             **mod_kwargs,
         )
 
+    @lazyval
+    def hits(self):
+        b = self.beatmap
+        rdata = self.actions
```

Comment:
yeah this is left over from before i put this code into the library (i really should check these things).

i was using replay_data because i think that's generally what it's called in the official osu info ([here](https://github.com/ppy/osu-api/wiki#get-replay-data)) but then again i don't think that name really makes much sense, i'd rather stick with actions